test/1_check_dependencies.sh

```diff
@@ -26,3 +26,11 @@ elif [[ "$PLATFORM" = "kubernetes" ]]; then
   check_env_var "GCLOUD_ZONE"
   check_env_var "GCLOUD_PROJECT_NAME"
 fi
+
+if [[ "$PLATFORM" == "kubernetes" ]] && ! is_minienv; then
```

Comment:
I am actually not so sure we even need the mini_env because I don't think we support it. What say you if I just remove these 2 blocks and add the DOCKER_REGISTRY_URL to the above else statement?

Something like:
```
elif [[ "$PLATFORM" = "kubernetes" ]]; then
  check_env_var "GCLOUD_SERVICE_KEY"
  ...
  **check_env_var "DOCKER_REGISTRY_URL"**
fi
```

Reply:
if we don't need `minienv` then it makes sense. why don't we need it? why is it needed in `kubernetes-conjur-demo`?

Reply:
I am not sure why in `demo` they decided to use `mini_env` as a test env and I see that is also something also mentioned in `deploy`. At least for our purposes, there is no mention of `MINI_ENV` in our project. Removing